compose/runtime/runtime/src/wasmJsTest/kotlin/androidx/compose/runtime/Actuals.wasmJs.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.runtime
+
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.await
+import kotlinx.coroutines.promise
+import kotlinx.coroutines.test.TestResult
+
+internal actual suspend fun TestResult.awaitCompletion() {
+    this.await<JsAny>()
```

Comment:
@eymar It doesn't work since `kotlinx.coroutines` 1.9.0 because now `TestResult` is typealiased into a separate external promise declaration